lib/msg.rb

```diff
@@ -0,0 +1,34 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Msg
+  attr_reader :delivery_time, :credentials
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def expected_time
+    @delivery_time = (Time.new + 3600).hour.to_s + ":" + (Time.new + 3600).min.to_s
+  end
+
+  def send_confirmation
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for send_confirmation is too high. [15.81/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [14/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)